pybossa/view/admin.py

```diff
@@ -577,6 +579,20 @@ def management_dashboard():
                            task_chart=task_chart,
                            submission_chart=submission_chart)
 
+@blueprint.route('/usage_dashboard/')
+@login_required
+@admin_required
+def usage_dashboard():
+    str_months = str(request.args.get('months'))
```

Comment:
```request.args.get``` returns "string" by default.
I think the better way is the following one line to replace line 586 to 588:
```
months = request.args.get(key='months', default=12, type=int)
```
You will need to modify line 590 accordingly if use the single line above.

Reply:
A small nit with this method is the url does not change. I think its more transparent to the user to see the page get redirected to a url with correct arguments, but that's just me.

Reply:
Could a default value be set to 12-months without a redirect?

Reply:
Either method makes sense. I tend to agree that for unique url safety, the redirect may provide an advantage (especially for search and indexing of urls to indicate an invalid argument will redirect back to a proper default).